.github/workflow/actions.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Makefile with Selenium
+
+on:
+  # Schedule to run every 2 minutes
+  schedule:
+    - cron: '*/2 * * * *'
```

Comment:
We want to run it on a monthly basis, eg, on the 1st day of each month.

Reply:
Fixed